impress-org/givewp
# DonorProfiles domain namespace is now DonorDashboards

**_NOTE: Changes in this PR are dependent upon changes in these PRs:_**
- https://github.com/impress-org/give-annual-receipts/pull/92
- https://github.com/impress-org/give-recurring/pull/1038

Resolves #5726 

## Description

This PR introduces changes to ensure that the DonorProfiles  domain namespace is instead referred to as DonorDashboards.

## Affects

This PR affects all references to the DonorProfiles namespace.

## Visuals

N/A

## Testing Instructions

1. Make sure you are testing with all the dependent PRs
2. Check various functionality of the DonorDashboard domain, to ensure that there are no glaring breaks

## Pre-review Checklist
